annotations/static/annotations/js/annotators/text.js

```diff
@@ -93,7 +140,7 @@ var ConceptSearch = {
                 payload['pos'] = this.pos;
             }
             if (this.force) {
-                payload['force'] = 'force';
+                payload['force'] = this.force;
```

Comment:
shouldn't ```payload['force']``` always be what ```this.force``` is?

Reply:
Thats what I would think. The if statement was already there so I assumed it was wanted for some reason. The false case was just handled as none. Ill remove it.